### DISCUSSION TITLE
Jetson Nano Setup

### DISCUSSION THREAD

**Discussion by mitchross** [tomshaffner/PiThermalCam#6] [General]
This project is fully compatible with jetson nano with a few small tweaks.

Python Dependencies
For matplotlib, I had to use this to install.
`sudo apt-get install python-matplotlib
`

First edit your .bashrc file and add the line `export OPENBLAS_CORETYPE=ARMV8` 
This will resolve any "Illegal instruction (core dumped)" issues.
 Dont forget to `source .bashrc` for effects to take place 

Next change the i2c clock rate from 100000 to 1000000 ( 1 extra 0 )
`sudo nano /sys/bus/i2c/devices/i2c-1/bus_clk_rate`

Everything should work from here.

**Comment by tomshaffner** [comment]
Great addition; thanks for adding the instructions!!

**Reply by mitchross** [reply]
You may want to consider refactoring the web server code to not have hard coded directories. Maybe you can use CWD in python? 
![image](https://user-images.githubusercontent.com/6330506/120672638-7d1eb100-c460-11eb-9475-233487a53e55.png)
